src/webrtc/vad.md

```diff
@@ -0,0 +1,66 @@
+# Active Speaker Detection
+
+A common feature in video conferencing software is active speaker detection.
+It is a small feature, but one that the user would feel weird without.
+Oftentimes, it is visualised by highlighting the user's tile while they are talking.
+Other use cases include deciding which participants should be shown on the screen.
+This can be implementedby using historical data about their voice activity.
+
+As you can see, voice activity detection is a small, but important piece of any WebRTC implementation.
+WebRTC standard provides tools that make it easy and convenient to implement this in your Selective Forwarding Unit.
+
+## Audio Level Header extension
+RFC 6464 defines an Audio Level Header extension, which is very useful in the context of Voice Activity Detection, as it takes the implementation load of the SFU.
+
+The Extension carries the information about the audio level in `-dBov` with values from 0 to 127.
+Please pay attention to the minus, as it makes it so that the louder it gets, the lower the value becomes.
+The fact that values are negative is a consequence of a definition of the `dBov` unit.
+
+RFC 6464 also defines an optional flag bit "V".
+When the use of it is negotiated, it indicates whether the encoder believes the audio packet contains voice activity.
+
+### What is `dBov`
+`dBov` is defined as the level (in [decibels](https://en.wikipedia.org/wiki/Decibel)) relative to the overload point of the system.
+In this context, an overload point is the highest-intensity signal encodable by the payload format.
+In simpler terms, the overload point is the loudest possible sound that can be encoded into the codec.
+
+## Audio Level processing
+You could use the flag bit "V" if available, but we don't recommend using it for the production environment for 2 reasons:
+1. It is optional, so you cannot be sure it will always be there.
+2. Implementation isn't standardized, meaning that you can have inconsistent behavior depending on the sender.
+
+It is therefore worth considering implementing your algorithm based purely on the `level` field.
+Hopefully, you now have some understanding of the value in the `level` field, and we can jump right into the algorithm.
+Don't worry, it's not difficult at all.
```

Comment:
I wonder if this sentence wouldn't fit better at the end of `Audio Level Header extension`